examples/techdocs/wayback-search/docs/index.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+This is an example of what you can do in techdocs.
+
+Here is the overview page for the wayback search component.
+
+It's cool isn't it?
+
+# Assets, links
+
+Look at this lovely cat!
+
+![This is a pretty cute cat](./cute-cat.png)
+
+We can also link you to a lovely video of a cat: [this is a lovely cat video](https://www.youtube.com/watch?v=cbP2N1BQdYc)
+
+
+# Relationship diagrams
+
+```plantuml format="png" classes="uml myDiagram" alt="Roadie" title="My super diagram" width="300px" height="300px"
```

Comment:
Maybe headers for these diagrams. 
'Look at this! Built with plantUML' and possible the actual code behind it also (if possible, a collapsable item)

Reply:
Sure thing.